cli/src/def/build.ts

```diff
@@ -62,6 +62,11 @@ export abstract class BaseBuildCommand extends Command {
       'Path and filename of generated type def file. Relative to `--output_dir`',
   })
 
+  noConstEnum?: boolean = Option.Boolean('--no-const-enum', {
```

Comment:
```suggestion
  constEnum?: boolean = Option.Boolean('--const-enum', {
```

and the `--no-const-enum` option will be auto generated

Reply:
🤯 I didn't know it generated `--no-const-enum`. I googled for a very long time.
I tried `--const-enum=false`, `--const-enum=0`, `--const-enum false`. 

It would be great to document this in the CLI usage help. Is there a way to automatically generate the negated documentation as well, or should I add it manually?